llama_index/llms/ollama.py

```diff
@@ -63,7 +63,7 @@ def metadata(self) -> LLMMetadata:
     def _model_kwargs(self) -> Dict[str, Any]:
         base_kwargs = {
```

Comment:
do we need to change "option" here for the model to include `**self.additional_kwargs`

Reply:
Since all the model related additional_kwargs need to put under `options` key for ollama, it seems more reasonable to me to just put them under that key when calling the api.

Reply:
But what if we want to pass parameters like `template`, `system`? With this change, it throws an error for the code below. For eg.

```
from llama_index.llms.ollama import Ollama

llm = Ollama(
    model='mistral-7b:latest', 
    additional_kwargs={
        "system": "You are a pirate with a colorful personality"
    })

response = llm.stream_complete("Hello there!")
for r in response:
    print(r.delta, end="")
```

I think the previous implementation was better and provided better clarity.

Reply:
> But what if we want to pass parameters like `template`, `system`? With this change, it throws an error for the code below. ...

This was fixed in https://github.com/run-llama/llama_index/pull/9558
I apologize for my carelessness